src/issue-strategies/bug-maintenance.js

```diff
@@ -1,7 +1,18 @@
+import * as jiraOperations from '../jira-operations.js';
```

Comment:
We only need `findParent`, so `* as jiraOperations` is a bit much. Also, the `.js` extension is unnecessary.